src/index.html

```diff
@@ -17,6 +30,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
```

Comment:
The `href` attribute for the logo link is set to `index.html`. If this file is already `index.html`, this link will not navigate anywhere. Consider updating this link to point to the homepage or another relevant page.